packages/server/tests/acceptance/ws/subscribe.spec.ts

```diff
@@ -220,6 +220,19 @@ describe('@web-socket Acceptance Tests', async function() {
             webSocket.close();
         });
 
+        it('Connection TTL is enforced, should close all connections', async function() {
+            const wsConn2 = await new ethers.providers.WebSocketProvider(WS_RELAY_URL);
+
+            const wsConn3 = await new ethers.providers.WebSocketProvider(WS_RELAY_URL);
+            await new Promise(resolve => setTimeout(resolve, 300)); // Wait for the connections to be established
+
+            expect(server._connections).to.equal(3);
```

Comment:
The test itself on add 2 connections.
Add a comment to note that it includes `wsProvider` created in the the `beforeEach()`

Reply:
✅  Done! 👍